feature-account-api/src/main/java/io/novafoundation/nova/feature_account_api/data/fee/capability/CustomFeeAvailabilityFacade.kt

```diff
@@ -0,0 +1,13 @@
+package io.novafoundation.nova.feature_account_api.data.fee.capability
+
+import io.novafoundation.nova.runtime.multiNetwork.chain.model.Chain
+
+interface CustomFeeCapability {
+
+    suspend fun canPayFeeInNonUtilityToken(chainAsset: Chain.Asset): Boolean
+}
+
+interface CustomFeeCapabilityFacade {
+
+    suspend fun canPayFeeInNonUtilityToken(chainAsset: Chain.Asset, customFeeCapability: CustomFeeCapability): Boolean
+}
```

Comment:
With current namings its a bit unclear what is the difference between them
I understand the difference itself from correpsponding implementations and overall idea to split per-chain specific logic and general one realted to proxies is good, its just naming that makes difference unclear

Reply:
Yes, indeed, the name isn't clear enough, but it seems to me that everything becomes clear quite quickly if you open the implementation of the facade. I think we can leave it like this